stripe-crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/viewmodels/OnrampCoordinatorViewModel.kt

```diff
@@ -24,7 +32,9 @@ import kotlinx.coroutines.launch
  */
 internal class OnrampCoordinatorViewModel(
     private val handle: SavedStateHandle,
-    private val linkController: LinkController
+    private val linkController: LinkController,
+    private val cryptoApiRepository: CryptoApiRepository,
+    private val onrampCallbacks: OnrampCallbacks
 ) : ViewModel() {
```

Comment:
let's make this injectable. A good reference of how to do this is [LinkControllerViewModel](https://github.com/stripe/stripe-android/blob/master/paymentsheet/src/main/java/com/stripe/android/link/LinkControllerViewModel.kt#L42)